lib/httplib/httplib.go

```diff
@@ -152,11 +151,8 @@ func ConvertResponse(re *roundtrip.Response, err error) (*roundtrip.Response, er
 	if err != nil {
 		var uErr *url.Error
 		if errors.As(err, &uErr) && uErr.Err != nil {
-			var hostnameErr x509.HostnameError
-			if errors.As(uErr.Err, &hostnameErr) {
-				return nil, trace.ConnectionProblem(uErr.Err, uErr.Error())
-			}
-			return nil, trace.ConnectionProblem(uErr.Err, "error parsing URL")
+			// return nil, trace.ConnectionProblem(uErr.Err, "%s %q: %s", uErr.Op, uErr.URL, uErr.Err)
+			return nil, trace.ConnectionProblem(uErr.Err, "")
```

Comment:
```suggestion
			return nil, trace.ConnectionProblem(uErr.Err, "")
```

Are we ok with dropping the `url.Error` wrapper here? Is that what this PR is about?

Reply:
This PR is basically a revert from the previous PR which would report a generic "error parsing URL" message, but in a way that won't break the formatting if sending in URL strings with misinterpreted escaped chars.